### PR TITLE
update backup and restore with 2.4 changes

### PR DESCRIPTION
**Pull request by prat98**

### CODE REVIEW

File: stories/topics/proc-aws-generate-restore-data-file.adoc [hunk 1]
```diff
@@ -17,23 +17,25 @@ $ mkdir command_generator_data
 ----
 $ docker run --rm -v $(pwd)/command_generator_data:/data $IMAGE \
   command_generator_vars aws_restore_stack \
-  --output-data-file /data/extra_vars.yml
+  --output-data-file /data/extra_vars.yaml
```
Comment (zkayyali812) [inline stories/topics/proc-aws-generate-restore-data-file.adoc:20]: Keep all examples as .yml for consistency

File: stories/topics/proc-aws-generate-restore-data-file.adoc [hunk 2]
```diff
@@ -17,23 +17,25 @@ $ mkdir command_generator_data
...
 ----
 
-. After running these commands, a `command_generator_data/extra_vars.yml` template file is created. 
+. After running these commands, a `command_generator_data/extra_vars.yaml` template file is created.
```
Comment (zkayyali812) [inline stories/topics/proc-aws-generate-restore-data-file.adoc:23]: same

File: stories/topics/proc-aws-run-restore-playbook.adoc [hunk 3]
```diff
@@ -9,29 +9,31 @@ The following procedure runs the restore playbook as a container.
 +
 [literal, options="nowrap" subs="+attributes"]
 ----
-$ docker run --rm -v $(pwd)/command_generator_data:/data $IMAGE command_generator aws_restore_stack --data-file /data/extra_vars.yml
+$ docker run --rm -v $(pwd)/command_generator_data:/data $IMAGE command_generator aws_restore_stack --data-file /data/extra_vars.yaml
```
Comment (zkayyali812) [inline stories/topics/proc-aws-run-restore-playbook.adoc:12]: same

File: stories/topics/proc-aws-run-restore-playbook.adoc [hunk 4]
```diff
@@ -9,29 +9,31 @@ The following procedure runs the restore playbook as a container.
...
+$ docker run --rm --env PLATFORM=AWS -v ~/.aws/credentials:/home/runner/.aws/credentials:ro --env ANSIBLE_CONFIG=../aws-ansible.cfg \
+--env DEPLOYMENT_NAME=AnsibleAutomationPlatform --env GENERATE_INVENTORY=true --env CHECK_GENERATED_INVENTORY=false  \
+$IMAGE redhat.ansible_on_clouds.aws_restore_stack -e 'aws_foundation_stack_name=AnsibleAutomationPlatform \
+aws_backup_name=ansible-automation-platform-bucket-20230706T163309 aws_region=us-east-1 \
+aws_s3_bucket=ansible-automation-platform-bucket aws_ssm_bucket_name=aap-ssm-bucket'
 
 ----
 . The playbook can take some time to create a restored deployment.
-After you have run the playbook successfully, a restored deployment is available in AWS CloudFormation matching the name provided for it in the `extra_vars.yml` file.
+After you have run the playbook successfully, a restored deployment is available in AWS CloudFormation matching the name provided for it in the `extra_vars.yaml` file.
```
Comment (zkayyali812) [inline stories/topics/proc-aws-run-restore-playbook.adoc:39]: same

**Review by zkayyali812 — CHANGES_REQUESTED** [review]

**Review by ianf77 — APPROVED** [review]
LGTM

**Pull request merged**